### PR TITLE
adding port number for outgoing configuration

### LINKED ISSUE #1472: Bad reporting of OPENGROK_WEBAPP_CFGADDR value
Hello,

While testing 0.13-rc10, I set variable OPENGROK_WEBAPP_CFGADDR to 'localhost:2425'.

When indexng, I see in the log:
12:04:09 INFO: Done indexing data of all repositories (took 7.919 seconds)
12:04:09 INFO: Sending configuration to: **localhost**
12:04:09 INFO: Configuration update routine done, check log output for errors.

Could you fix to get the correct value:
12:04:09 INFO: Sending configuration to: **localhost:2425**

### PR DESCRIPTION
**Pull request by tulinkry**
fixes #1472

### CODE REVIEW

File: src/org/opensolaris/opengrok/index/Indexer.java [hunk 1]
```diff
@@ -869,7 +869,7 @@ public void refreshSearcherManagers(RuntimeEnvironment env, List<String> project
     }
 
     public void sendToConfigHost(RuntimeEnvironment env, String host, int port) {
-        LOGGER.log(Level.INFO, "Sending configuration to: {0}", host);
+        LOGGER.log(Level.INFO, "Sending configuration to: {0}:{1}", new Object[]{host, Integer.toString(port)});
```
Comment (vladak) [inline src/org/opensolaris/opengrok/index/Indexer.java]: there is one instance w/ missing port # couple of lines below

Comment (tulinkry) [inline src/org/opensolaris/opengrok/index/Indexer.java]: Fixed.

File: src/org/opensolaris/opengrok/index/Indexer.java [hunk 2]
```diff
@@ -869,12 +869,12 @@ public void refreshSearcherManagers(RuntimeEnvironment env, List<String> project
...
         try {
             env.writeConfiguration(host, port);
         } catch (IOException ex) {
             LOGGER.log(Level.SEVERE, "Failed to send configuration to "
-                    + host + " (is web application server running with opengrok deployed?)", ex);
+                    + host + ":" + port + " (is web application server running with opengrok deployed?)", ex);
```
Comment (vladak) [inline src/org/opensolaris/opengrok/index/Indexer.java]: use same formatting like the above print ?

Comment (tulinkry) [inline src/org/opensolaris/opengrok/index/Indexer.java]: Not possible. Achieved with String.format method.

**Pull request merged**